.eslintrc.js

```diff
@@ -390,6 +396,7 @@ module.exports = {
 			rules: {
 				'no-restricted-syntax': [
 					'error',
+					...restrictedSyntax,
```

Comment:
We also need to add `...restrictedSyntaxComponents` here, but since it will catch a bunch of previously uncaught violations, I'll address this in a separate PR.